packages/docusaurus-theme-search-algolia/src/theme/SearchPage/index.js

```diff
@@ -294,7 +294,7 @@ function Search() {
   }, [searchValue]);
 
   return (
-    <Layout title={getTitle()}>
+    <Layout title={getTitle()} wrapperClassName="search-results-wrapper">
```

Comment:
`search-page-wrapper` is better suited because "results" can only refer to a specific section of this page.